app/views/email_authentications/invitations/new.html.erb

```diff
@@ -1,12 +1,33 @@
 <h1><%= t "devise.invitations.new.header" %></h1>
+<% if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) %>
+  <div class="row">
+    <div class="col-md-6">
+      <%= bootstrap_form_with(url: email_authentication_invitation_path, local: true, autocomplete: "off", label_errors: true, html: {onsubmit: "submitForm(this)"}) do |form| %>
+        <%= form.text_field :full_name, id: :full_name, required: true, autocomplete: "off", label: "Full Name *" %>
+        <%= form.text_field :email, id: :email, required: true, label: "Email *" %>
+        <%= form.text_field :role, id: :role, required: true, help: "CVHO, STS, State Official etc.", label: 'Job Title *' %>
+        <%= form.select :access_levels, current_admin.permitted_access_levels_info, id: :access_levels, required: true, label: "Access *" %>
 
-<%= react_component "InviteAdminForm",
-                    organizations: policy_scope([:manage, :admin, Organization.all]),
-                    facility_groups: policy_scope([:manage, :admin, FacilityGroup.all]).order(:name),
-                    facilities: policy_scope([:manage, :admin, Facility.all]).pluck(:id, :name),
-                    permissions: policy([:manage, :admin, User]).allowed_permissions,
-                    access_levels: policy([:manage, :admin, User]).allowed_access_levels,
-                    submit_route: '/email_authentications/invitation',
-                    allow_email_edit: true,
-                    submit_text: 'Invite Admin',
-                    submit_method: 'POST' %>
\ No newline at end of file
+        <%= form.label "Facility Access *" %>
+        <div id="facility-access">
+          <%= render "email_authentications/invitations/access_tree",
+              access_tree: current_admin.access_tree,
+              form: form %>
+        </div>
+        <%= form.hidden_field :selected_facilities, id: "facility-access-selections", value: [], multiple: true %>
```

Comment:
This thing is magically populated by JS to mimic the conventional Rails Way™ of dealing with collections. Since this is not a nested form, these facilities are populated by multiple hidden tags under `selected_facilities` through [this](https://github.com/simpledotorg/simple-server/pull/1222/files#diff-be260484baa204aaf798736297ef46acR19) function, which then gets read as a well-formed array in the controller when the form is submitted.

Reply:
Thanks. It really isn't, I did that because it was annoying to figure out how collection checkboxes worked. I've implemented your suggestion [here](https://github.com/simpledotorg/simple-server/pull/1222/commits/b12ec7964ea60a3dc8c33970b6d736c52521bdf0). Works well and shaves off some JS ❇️ .